doc/whats_new.rst

```diff
@@ -15,6 +15,9 @@ Current
 Changelog
 ~~~~~~~~~
 
+- :func:`read_raw_bids` will return the the raw object with :code:`raw.info['bads']` already populated, whenever a :code:`channels.tsv` file is present, by `Stefan Appelhoff`_ (`#209 <https://github.com/mne-tools/mne-bids/pull/209>`_)
+- :func:`read_raw_bids` no longer optionally returns :code:`events` and :code:`event_id` but returns the raw object with :code:`mne.Annotations`, whenever an :code:`events.tsv` file is present, by `Stefan Appelhoff`_ (`#209 <https://github.com/mne-tools/mne-bids/pull/209>`_)
+
```

Comment:
This change would break backwards compatibility, no?

Reply:
yes, but I remember discussions we had over this some months ago, when @monkeyman192 also noted that it might be a bad idea to have a function optionally return 1, or 3 parameters

And together with the new `mne.Annotations` this API change is definitely "worthwhile" IMHO